src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,50 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends HomePage {
+
+    public LoginPage(WebDriver driver) {
+        super(driver);
+    }
+
+    private By emailFieldForSignUp = By.xpath("//input[@id='email_create']");
+    private By createAccountButton = By.xpath("//button[@id='SubmitCreate']");
+    private By emailFieldForSignIn = By.xpath("//input[@id='email']");
+    private By passwordFieldForSignIn = By.xpath("//input[@id='passwd']");
+    private By signInButton = By.xpath("//button[@id='SubmitLogin']");
+    private By invalidEmailError = By.xpath("//li[text()='Invalid email address.']");
```

Comment:
ammm. why u spicify error text here? why not to take a base element and text from it?